ui/src/app/base/sagas/actions.ts

```diff
@@ -85,28 +88,20 @@ export function* createPoolWithMachines(
  */
 export const generateNextUpdateRecordAction = (
   params: UpdateRecordParams
-): (() => WebSocketAction)[] => [
-  () => {
-    const { domain, name, resource, rrdata, ttl } = params;
-    if (isAddressRecord(resource.rrtype)) {
-      return domainActions.updateAddressRecord({
-        address_ttl: ttl,
-        domain,
-        ip_addresses: rrdata?.split(/[ ,]+/) || [],
-        name,
-        previous_name: resource.name,
-        previous_rrdata: resource.rrdata,
-      });
-    }
-    return domainActions.updateDNSData({
-      dnsdata_id: resource.dnsdata_id,
-      dnsresource_id: resource.dnsresource_id,
-      domain,
-      rrdata,
-      ttl,
-    });
-  },
-];
+): (() => WebSocketAction)[] => {
+  const { domain, name, rrset } = params;
+  if (!isAddressRecord(rrset.rrtype) && name !== rrset.name) {
```

Comment:
It might be helpful to leave a comment here about why it might not need to provide a next action.